### PR TITLE
feat: show a link to the comparison range on bisect complete

### PR DESCRIPTION
**Pull request by nornagon**
For easy clickin'!

Screenshot_9_18_20__10_43_AM

### CODE REVIEW

**Comment by codebytere** [comment]
@nornagon do you have a screenshot of the new UI?

**Review by codebytere — COMMENTED** [review]
@nornagon do you have a screenshot of the new UI?

This is also struggling here:

```
FAIL tests/renderer/components/commands-spec.tsx
  ● Test suite failed to run
    src/renderer/components/commands-bisect.tsx:49:9 - error TS2322: Type 'Element' is not assignable to type 'string'.
    49         label: <>Bisect complete. {message}</>,
               ~~~~~
      src/interfaces.ts:60:3
        60   label: string;
             ~~~~~
        The expected type comes from property 'label' which is declared here on type 'GenericDialogOptions'
```

**Comment by nornagon** [comment]
@codebytere screenshot added to description & tests fixed.

**Review by codebytere — APPROVED** [review]

File: src/renderer/components/commands-bisect.tsx [hunk 1]
```diff
@@ -29,12 +29,24 @@ export class BisectHandler extends React.Component<BisectHandlerProps> {
...
+          </a>
+          .
+        </>
+      );
 
       appState.pushOutput(`[BISECT] Complete: ${message}`);
       appState.setGenericDialogOptions({
         type: GenericDialogType.success,
-        label: `Bisect complete. ${message}`,
+        label: <>Bisect complete. {message}</>,
```
Comment (HashimotoYT) [inline src/renderer/components/commands-bisect.tsx:49]: I think for more organizing we can move this "Bisect complete." also inside the `message` constant, and just return the single constant with all inside. This also should help to remove the second `React.Fragment`.

**Review by HashimotoYT — APPROVED** [review]
Looks cool, one small improvement.

**Pull request merged**